public-images/public-images-how-to/run-an-ova-using-virtualbox.rst

```diff
@@ -0,0 +1,258 @@
+ISO image.
+
+1. Make a YAML file with password authorisation enabled and your public
+   SSH key. Replace ``<YOUR_PUB_KEY>`` with your public key.
+
+.. code:: bash
+
+  cat <<EOF > my-cloud-config.yaml
+  #cloud-config
+  chpasswd:
+    list: |
+      ubuntu:ubuntu
+  expire: False
+  ssh_pwauth: True
+  ssh_authorized_keys: <YOUR_PUB_KEY>
+  EOF
+
+2. Make the ISO.
+
+.. code:: bash
+
+  cloud-localds my-seed.iso my-cloud-config.yaml
+
+3. Attach the ISO as an optical drive.
+
+  a. From the VirtualBox main menu, select the image on the left, right
+     click and choose **Settings.**
+
+  b. Select **Storage** from the menu options.
+
+  .. image:: run-an-ova-using-virtualbox-images/4-add-optical.png
+     :align: center
+
+  c. Click on the +CD icon (**Adds Optical Drive**) beside **Controller:
+     IDE.**
+
+  .. image:: run-an-ova-using-virtualbox-images/5-select-seed.png
+     :align: center
+
+  d. Click **Add** and select the ``my-seed.iso`` file we created above, then
+     click **Choose**.
+
+We are now ready to launch the VM.
+
+Launching the VM
+----------------
+
+From the VirtualBox main menu, click **Start**. This will open a new
+window and begin the launch process for your VM. This may take some
+time.
+
+.. image:: run-an-ova-using-virtualbox-images/6-vm-login-prompt.png
+   :align: center
+
+When you reach the ``ubuntu login:`` prompt, enter the username and
+password from the YAML file above. If you didn’t change the template,
+both the username and password will be ``ubuntu``.
+
+.. image:: run-an-ova-using-virtualbox-images/7-vm-logged-in.png
+   :align: center
+
+Success! Now is a good time to run some commands to ensure everything is
+set up correctly.
+
+-  Verify that you can access the internet.
+
+.. code:: bash
+
+   ping -c 3 ubuntu.com
+
+-  Check your IP configuration (needed for connecting via SSH).
+
+.. code:: bash
+
+   ip a
+
+-  Verify that the SSH service is running.
+
+.. code:: bash
+
+   sudo systemctl status ssh
+
+-  Check disk configuration.
+
+.. code:: bash
+
+   df -h
+
+When you have finished with you session, you can turn off the VM by
+selecting **File** > **Close** from the top bar menu of the window
+running the machine. There are more options available under the
+**Machine** heading.
+
+Connecting via SSH
+------------------
+
+To connect via SSH, we will make use of the bridged network adapter
+that was configured during the appliance setup. You will need to have
+the IP address of the VM. Run the following from the virtual console of
+the VM.
+
+.. code:: bash
+
+  ip a
+
+Below is an example output of this command.
+
+.. terminal::
+
+  $ ip a
+    1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN
+    group default qlen 1000
+    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
+    inet 127.0.0.1/8 scope host lo
+    valid_lft forever preferred_lft forever
+  2: enp0s3: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel
```

Comment:
the indentation looks wrong here, just want to double check that it's what you expect.

Reply:
Good catch, I tried to emulate the `ip a` indents but they got jumbled when I shifted everything into the rst `terminal` block.